### ISSUE TITLE
Nie można dodać produktu do koszyka / Nie można utworzyć nowego użytkownika - blocked by CORS

### ISSUE THREAD

**Issue by mdlubak** [allegro/allegro-api#9108]
Występuje problem: 
 - [ ] z zasobem REST API / with a REST API resource
 - [ ] ze środowiskiem produkcyjnym / with the production environment
 - [X ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**

Przy próbie dodania produktu do koszyka przycisk wyszarza się w momencie kliknięcia, ale nie dodaje produktu do koszyka. 

![image](https://github.com/allegro/allegro-api/assets/24640670/1038937f-37bc-4ba3-90fc-dc6242538c69)

Taki sam błąd występuje przy próbie utworzenia nowego użytkownika, po kliknięciu 'Załóż konto':
![image](https://github.com/allegro/allegro-api/assets/24640670/88656f36-8740-44d7-b048-5b1ce5d7123f)

**Comment by AureliuszBrussy** [comment]
Czy po czyszczeniu plików cookies (lub w trybie incognito) problem wciąż występuje? Nie potwierdzam w tym momencie szerszych problemów z Sandboxiem i CORS.

**Comment by mdlubak** [comment]
Tak, po czyszczeniu cookies oraz w trybie incognito problem nadal występował. W tej chwili problem zniknął - mogę dodać produkty do koszyka.
